app/models/workarea/payment/credit_card.decorator

```diff
@@ -23,5 +23,10 @@ module Workarea
         end
       end
     end
+
+    def issuer_accepted
```

Comment:
Does this return a boolean? Should it end with `?`?

Reply:
#issuer_accepted is a method defined in core this way.